benchmark/image-classification/benchmark_resnet.py

```diff
@@ -61,7 +65,8 @@
 
 if args.data_set == "imagenet":
     if not args.train_mode or not args.data_path:
-        print("Need to provide training mode(train_on_batch or fit_generator) and data path to imagenet dataset")
+        print("Need to provide training mode(train_on_batch or fit_generator) "
```

Comment:
Lets use a logger for logging these - to be able to differentiate between debug & error logs - https://docs.python.org/2/howto/logging.html